templates/part-category/part-category.js

```diff
@@ -183,6 +191,14 @@ function setCanonicalUrl(category) {
   document.head.appendChild(canonical);
 }
 
+function resolvePartLabel(type, cat) {
```

Comment:
I'm okay with the short version of variable names, but please set it as `category` instead. Thanks.